Create masters-golf.md

gregpent opened the pull request:
Engaging article about cryptocurrency and golf

PhoenixGreen commented:
ZeroGPT score = 15%

gregpent commented:
> ZeroGPT score = 15%

Yes I was surprised with this and when I went to see what it picked up on in yellow, it basically picked up on my explanation of golf which I entirely wrote myself . So I’m not convinced entirely on the accuracy in certain parts. I’ll give it another review today to see if changing the commas or parts affects the score.

PhoenixGreen commented on masters-golf.md:
```diff
@@ -0,0 +1,105 @@
+* Who might be a good person to interview for this piece?
+
+
+
```
I've just finished reading this piece and I have a few thoughts and questions. Generally I think the golf use case for blockchain technology is quite interesting. But I think if we are going to report on this we need to make sure we cover the cypherpunk principles and ask unbiased questions. We also need to Digg deeper into the facts and produce evidence to backup our statements. Mainstream crypto media will always point in the direction of the positive but it's our job to scrutinise these views with counter arguments and evidence.

Firstly, it would have been good if you had put up an outline and thesis statement for this piece before writing. The reason for this is I'd like to start building our pieces around critical thinking and unbiased arguments. Asking basic questions like:

* Who does this new technology benefit? Who is aiming to profit?
* Does this new technology remove the middleman or just replace them?
* Is this new technology an improvement for the user or does it just digitise an already perfect system?
* Are these new systems going to be non-custodial and peer to peer or will they be controlled? If so, how?

As a final note to these questions, they should also be backed up with evidence e.g. This new system works in the following way ... which demonstrated that it is .... and here's my source.

From what I've read here it looks like the current systems of merchandise, handicaps grading, and scoring is being replaced by new centralised entities. In my opinion, this is decentralised in name only, with an aim to replace the current establishment with another. Do you see this differently? If so, please elaborate.

Do you think the systems being replaced are an improvement on the old ones? Do you see any potential issues with these new systems? If so, what issues do you see?

gregpent replied:
I've addressed some points in the advantages and disadvantages but I will also add more detail

PhoenixGreen commented on masters-golf.md:
```diff
@@ -0,0 +1,105 @@
+
+## Thesis statement
+
+This article explores the potential integration of blockchain technology into the Golf world
```
This isn't a thesis statement. One way of structuring a thesis statement is to think about it as two separate sentences.

What are you researching?
What question/s are you hoping to find the answer to?
for example:

The research sentence:
"Bitcoin's POW mechanism was a revolution that helped solve the double spend problems and gave us a means to create decentralised money."

The Question to answer sentence:
"Has this mechanism maintained these core values or has it been superseded by other more improved mechanisms?"

The full thesis statement:
Bitcoin's POW mechanism was a revolution that helped solve the double spend problems and gave us a means to create decentralised money. Has this mechanism maintained these core values or has it been superseded by other more improved mechanisms?

PhoenixGreen commented on masters-golf.md:
```diff
@@ -0,0 +1,105 @@
+
+
+## Pre-discussion, research, and questions
+
```
To help with the pre-discussion, here you can post a list of questions you're aiming to find the answers to. Make your questions open ended and aim to find at least one contrary points of view to your own. Finding opposite opinions helps us be less biased and leads to more critical thinking.

For example:
We may hate the current handicap grading system but what does it do well? It allows players to keep track of their score and scrutinise officials face to face if they try to increase or decrease it too heavily. In many respects, this is the most purest form of decentralisation.

How will a digital system / blockchain system make this worst? ...

gregpent replied:
I've updated some questions I'd hope to cover in the article

PhoenixGreen commented:
Sorry it seems I clicked the "close pull request" button by mistake. Hopefully, this article still works and you can see my comments, now I've re-opened it.

gregpent commented:
> Sorry it seems I clicked the "close pull request" button by mistake. Hopefully, this article still works and you can see my comments, now I've re-opened it.

Yes I can still see. I’ll work on addressing these points today. I understand the article might not address all points. It’s a bit of an experiment article to see what engagement it gets. It might not go with the cypherpunk theme either so I appreciate you allowing this.

I’ll address the valid points you have made and review it today.

gregpent commented on masters-golf.md:
I've updated this and changed the wording

OfficialCryptos commented:
@gregpent i finished the artwork for you today ;)
![golf decred](https://github.com/PhoenixGreen/Cypherpunktimes-articles/assets/158556488/17a3fc68-b8ca-466f-b5e5-6b52f249fa1b)

OfficialCryptos commented:
Finished the artwork 
![golf decred](https://github.com/PhoenixGreen/Cypherpunktimes-articles/assets/158556488/ffed8a44-15be-420f-8cef-8c3b4a82734d)
